packages/shared/src/components/cards/Card.tsx

```diff
@@ -43,7 +43,31 @@ export const ListCardTitle = classed(Title, 'mr-2');
 
 export const CardTextContainer = classed('div', 'flex flex-col mx-4');
 
-export const CardImage = classed(Image, 'rounded-xl h-40');
+interface CardImageProps extends Omit<ImageProps, 'className'> {
+  className?: {
+    image?: string;
+    wrapper?: string;
+  };
+}
+
+export const CardImage = ({
+  className,
```

Comment:
For safety, since the object is not using optional chaining on the implementation part.
```suggestion
  className = {},
```